opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -56,4 +55,22 @@
    * and others consider the caller to be the parent and the receiver to be the child)
    */
   <T> Span join(String operationName, T carrier);
+
+  interface SpanBuilder {
```

Comment:
I like it... since java has neither named params nor quick ad-hoc structs, builder patterns work well.